docs/sphinx_doc/user/abl_bndry_io.rst

```diff
@@ -0,0 +1,71 @@
+.. _amrwind-abl-bndry-io:
+
+AMR-Wind ABL Boundary I/O
+=========================
+
+Wind farm simulations typically require ABL inflow conditions. As
+such, a precursor ABL simulation is often performed to collect inflow
+conditions for the wind farm. AMR-Wind leverages NetCDF to collect ABL
+inflow variables at each time step and write the data to a file. This
+file can then be read during a wind farm simulation to populate the
+inflow.
+
+.. note::
+
+   - Currently, it is only possible to write `xlo` and `ylo` boundaries.
+
+   - Inflow conditions are linearly interpolated between output times.
+
+   - The time for the simulation that is reading the inflow file must be entirely contained within the inflow times.
+
+   - The simulation reading the inflow file must have the same grid resolution at the boundaries.
+
+
+Generating the inflow file from an ABL simulation
+-------------------------------------------------
+
+The following section can be added to the input file to generate an
+inflow file during an ABL simulation:
+
+.. code-block:: none
+
+   ABL.bndry_output_file = "bndry_file.nc"
+   ABL.bndry_output_planes = ylo xlo
+   ABL.bndry_output_start_time = 2.0
+   ABL.bndry_output_var_names = velocity temperature
+
+
+Using an inflow file in an ABL simulation
+-----------------------------------------
+
+The following section can be added to the input file to read an
+inflow file to populate the boundary conditions:
+
+.. code-block:: none
+
+   ABL.bndry_input_file = "../orig/bndry_file.nc"
+   ABL.bndry_input_var_names = velocity temperature
+
+
+Inflow file structure
+---------------------
+
+The inflow file is written by the NetCDF library in the following structure:
```

Comment:
Include output of `ncdump -h <file>` here?

Reply:
added